### PR TITLE
[libftdi1] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

File: ports/libftdi1/portfile.cmake [hunk 1]
```diff
@@ -36,6 +36,12 @@ vcpkg_cmake_install()
 
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/libftdi1)
 
+if(VCPKG_TARGET_IS_WINDOWS)
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib/pkgconfig" "${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig")
```
Comment (JackBoosY) [inline ports/libftdi1/portfile.cmake:40]: Why remove this?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/libftdi1/portfile.cmake (hunk 1 above)
Comment (autoantwort) [reply ports/libftdi1/portfile.cmake:40]: Because the dependency does not have pkgconfig files and `vcpkg_fixup_pkgconfig` will fail

Comment (JackBoosY) [reply ports/libftdi1/portfile.cmake:40]: As I said in another PR, can we move it from `Libs.Private` to (such as) `Libs` to fix this issue?

Comment (autoantwort) [reply ports/libftdi1/portfile.cmake:40]: See https://github.com/microsoft/vcpkg/pull/21704#pullrequestreview-819436226

Comment (ras0219-msft) [reply ports/libftdi1/portfile.cmake:40]: It would be much better to just find and fix the missing .pc files.

Comment (autoantwort) [reply ports/libftdi1/portfile.cmake]: Ok I have now added the missing pc file

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the fixes!

**Pull request merged**